src/exact.jl

```diff
@@ -0,0 +1,89 @@
+"""
+    ot_plan(c, μ, ν; kwargs...)
+
+Compute the optimal transport plan for the Monge-Kantorovich problem with source and target
+marginals `μ` and `ν` and cost `c`.
+
+The optimal transport plan solves
+```math
+\\inf_{\\gamma \\in \\Pi(\\mu, \\nu)} \\int c(x, y) \\, \\mathrm{d}\\gamma(x, y)
+```
+where ``\\Pi(\\mu, \\nu)`` denotes the couplings of ``\\mu`` and ``\\nu``.
+
+See also: [`ot_cost`](@ref)
+"""
+ot_plan
```

Comment:
Why defining this docstring like this? Isn't it redundant since a very similar docstring is present in the following functions?

Reply:
The main motivation is to explain what `ot_cost` and `ot_plan` are designed for without focusing on the special cases such as e.g. for univariate distributions or discrete 1D measures. The common practice is to only define docstrings for the special cases if they are really necessary and e.g. have special keyword arguments or return types.

Reply:
I see. Perhaps in another PR, we should reduce some of the repeated info in the other repeated functions.